barretenberg/cpp/pil/vm2/execution.pil

```diff
@@ -25,6 +25,7 @@ include "public_data_check.pil";
 include "update_check.pil";
 include "nullifier_check.pil";
 include "public_inputs.pil";
+include "tx.pil";
```

Comment:
Should we make TX the new entry point for generation? It's the highest level thing.

Reply:
yeah we can!